Source/Public/Update-BicepParameterFile.ps1

```diff
@@ -0,0 +1,75 @@
+function Update-BicepParameterFile {
+    [CmdletBinding()]
+    param (
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory, Position = 1)]
+        [string]$Path,
+
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Position = 2)]
+        [string]$BicepFile
```

Comment:
Same thing here, we should validate that the input to `-BicepFile` is a `.bicep` file.